File: app/src/main/java/vgame/ir/data/local/entity/DefaultNoteEntity.java
package vgame.ir.data.local.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.SerializedName;

@Entity(tableName = "default_note")
public class DefaultNoteEntity {

    @PrimaryKey(autoGenerate = true)
    @SerializedName("id")
    private int id;

    @SerializedName("text")
    private String text;

    @SerializedName("lesson_id")
    private int lesson_id;

    @SerializedName("course_id")
    private int course_id;

    @SerializedName("lesson_name")
    private String lesson_name;

    @SerializedName("chapter_name")
    private String chapter_name;

    public String getChapter_name() {
        return chapter_name;
    }

    public void setChapter_name(String chapter_name) {
        this.chapter_name = chapter_name;
    }

    public String getLesson_name() {
        return lesson_name;
    }

    public void setLesson_name(String lesson_name) {
        this.lesson_name = lesson_name;
    }

    public int getId() {
        return id;
    }

    public int getLesson_id() {
        return lesson_id;
    }

    public int getCourse_id() {
        return course_id;
    }

    public void setCourse_id(int course_id) {
        this.course_id = course_id;
    }

    public void setLesson_id(int lesson_id) {
        this.lesson_id = lesson_id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: app/src/main/java/vgame/ir/data/local/entity/DefaultNoteEntity.kt
package vgame.ir.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

import com.google.gson.annotations.SerializedName

@Entity(tableName = "default_note")
class DefaultNoteEntity {

    @PrimaryKey(autoGenerate = true)
    @SerializedName("id")
    var id: Int = 0

    @SerializedName("text")
    var text: String? = null

    @SerializedName("lesson_id")
    var lesson_id: Int = 0

    @SerializedName("course_id")
    var course_id: Int = 0

    @SerializedName("lesson_name")
    var lesson_name: String? = null

    @SerializedName("chapter_name")
    var chapter_name: String? = null
}